SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -111,65 +139,72 @@ etc...
     NetConnectionClassName="/Script/SpatialGDK.SpatialNetConnection"
     ```
 
-    > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
+    > These lines ensure that the GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
+    > If you ever need to run with default Unreal networking for workflow or validation reasons, you can switch networking in the Unreal Editor: from the Editor menu, select **Play** and from the drop-down menu un-check `Spatial Networking`. This setting is valid for Editor and command-line builds. It is stored in your game project's Unreal config file; `<GameRoot>\Config\DefaultGame.ini` under `\Script\EngineSettings.GeneralProjectSettings`.
```

Comment:
"select Play and" - I think they will just click the button which will start the game.
Maybe something like "press the drop down arrow next to the **Play** button"?